src/psyclone/tests/dynamo0p3_stubgen_test.py

```diff
@@ -844,3 +844,17 @@ def test_stub_stencil_multi():
         "field_4_stencil_dofmap")
 
     assert result2 in generated_code
+
```

Comment:
Please update the years in the Copyright to "2017-2020".

Reply:
Done.